src/Controller/Controller_Zapier.php

```diff
@@ -38,22 +39,35 @@ public function init(): void {
 	 *
 	 * @return array
 	 *
-	 * @throws \GFPDF_Vendor\Spatie\UrlSigner\Exceptions\InvalidExpiration
-	 * @throws \GFPDF_Vendor\Spatie\UrlSigner\Exceptions\InvalidSignatureKey
+	 * @throws \Exception
 	 * @since 6.1
 	 */
 	public function add_expanded_zapier_support( $body, $feed, $entry, $form ) {
 		$zapier = \GF_Zapier::get_instance();
 		$signer = new Helper_Url_Signer();
 		$model  = \GPDFAPI::get_mvc_class( 'Model_PDF' );
+		$logger = new Helper_Logger( 'gravity-pdf', 'Gravity PDF' );
+		$log    = $logger->get_logger();
 
 		foreach ( $form['gfpdf_form_settings'] as $settings ) {
 			$url = $model->get_pdf_url( $settings['id'], $entry['id'], false, false, false );
 
-			$body[ $zapier->get_body_key( $body, $settings['name'] . ' SIGNED PDF URL' ) ]       = $url;
-			$body[ $zapier->get_body_key( $body, $settings['name'] . ' SIGNED PDF URL WEEK' ) ]  = $signer->sign( $url, '+1 week' );
-			$body[ $zapier->get_body_key( $body, $settings['name'] . ' SIGNED PDF URL MONTH' ) ] = $signer->sign( $url, '+1 month' );
-			$body[ $zapier->get_body_key( $body, $settings['name'] . ' SIGNED PDF URL YEAR' ) ]  = $signer->sign( $url, '+1 year' );
+			$body[ $zapier->get_body_key( $body, $settings['name'] . ' PDF URL' ) ] = $url;
+			try {
+				$body[ $zapier->get_body_key( $body, $settings['name'] . ' SIGNED PDF URL WEEK' ) ]  = $signer->sign( $url, '+1 week' );
+				$body[ $zapier->get_body_key( $body, $settings['name'] . ' SIGNED PDF URL MONTH' ) ] = $signer->sign( $url, '+1 month' );
+				$body[ $zapier->get_body_key( $body, $settings['name'] . ' SIGNED PDF URL YEAR' ) ]  = $signer->sign( $url, '+1 year' );
+			} catch ( \Exception $e ) {
+				$log->error(
+					'Generating Zapier signed PDF URLs error.',
+					[
+						'entry'     => $entry,
+						'settings'  => $settings,
+						'exception' => $e->getMessage(),
+					]
+				);
+				throw new \Exception( 'Zapier signed PDF URLs error.' );
```

Comment:
Remove this line. You just caught the Exceptions and handled them appropriately. No need to throw a new error.